zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/multitenancy/TenantAwareResourceFetchTest.java

```diff
@@ -66,31 +77,25 @@ final var record =
             .fetch();
 
     // then
-    ResourceAssert.assertThat(record.getValue())
-        .isNotNull()
-        .hasResourceProp(new String(resourceBytes))
-        .hasResourceKey(resourceKey)
-        .hasResourceId("Rpa_0w7r08e")
-        .hasResourceName("test.rpa")
-        .hasVersion(1)
-        .hasVersionTag("v1.0")
-        .hasDeploymentKey(deployment.getKey())
-        .hasChecksum(resourceMetadata.getChecksum())
-        .hasTenantId(TENANT_B);
-    verify(mockCommandResponseWriter).intent(ResourceIntent.FETCHED);
-    verify(mockCommandResponseWriter).valueType(ValueType.RESOURCE);
-    verify(mockCommandResponseWriter).key(resourceKey);
-    ResourceAssert.assertThat(resourceResponse)
-        .isNotNull()
-        .hasResourceProp(new String(resourceBytes))
-        .hasResourceKey(resourceKey)
-        .hasResourceId("Rpa_0w7r08e")
-        .hasResourceName("test.rpa")
-        .hasVersion(1)
-        .hasVersionTag("v1.0")
-        .hasDeploymentKey(deployment.getKey())
-        .hasChecksum(resourceMetadata.getChecksum())
-        .hasTenantId(TENANT_B);
+    verify(mockCommandResponseWriter, VERIFICATION_TIMEOUT).intent(ResourceIntent.FETCHED);
+    verify(mockCommandResponseWriter, VERIFICATION_TIMEOUT).valueType(ValueType.RESOURCE);
+    verify(mockCommandResponseWriter, VERIFICATION_TIMEOUT).key(resourceKey);
```

Comment:
FYI: I noticed that the tests were a bit flaky and sometimes failing when no timeout was specified.